.gitignore

```diff
@@ -52,3 +52,6 @@ nginx
 *.plist
 a.patch
 Makefile
+autoconf.err
+autotest
+objs/*
```

Comment:
Will you remove these? Seems like you are not using the standard building process based on `util/build.sh`.